docs/source/guides/session_limits.rst

```diff
@@ -0,0 +1,75 @@
+How To Set Orchestrator Session Limits
+======================================
+
+What Are Orchestrator Sessions?
+******************************
+
+By default an Orchestrator node can handle a maximum of **10** incoming livestreams at a time. Whenever the number of concurrent transcoding sessions go above this, the Orchestrator returns an ``OrchestratorCapped`` error to the Broadcaster node.
```

Comment:
Since we don't capitalize orchestrator/broadcaster elsewhere at the moment I think we should stick to that convention here.